core/pom.xml

```diff
@@ -177,10 +177,16 @@
         </dependency>
         <dependency>
           <groupId>org.hibernate</groupId>
-          <artifactId>hibernate-validator-cdi</artifactId>
+          <artifactId>hibernate-validator</artifactId>
```

Comment:
after clone repo and run mvn clean install,  tests complains about no provider found on classpath, besides maven complain about redeclare hibernate-validator-cdi saying its ignoring on it. Some idea ?

Reply:
I can't reproduce this problem on my machine. Which tests are making trouble without these changes?

Reply:
`Apache Maven 3.5.4 (1edded0938998edf8bf061f1ceb3cfdeccf443fe; 2018-06-17T15:33:14-03:00)
Maven home: D:\softwares\apache-maven-3.5.4\bin\..
Java version: 1.8.0_191, vendor: Oracle Corporation, runtime: C:\Program Files\Java\jdk1.8.0_191\jre
Default locale: pt_BR, platform encoding: Cp1252
OS name: "windows 10", version: "10.0", arch: "amd64", family: "windows"`

[ERROR] Errors:
[ERROR]   ConstraintViolationsMvcBindingTest.initValidator:60 » NoProviderFound Unable t...
[ERROR]   ConstraintViolationsTest.shouldSupportBeanParamBeans:98->simulateValidation:109 » NoProviderFound
[ERROR]   ConstraintViolationsTest.shouldSupportControllerFields:57->simulateValidation:109 » NoProviderFound
[ERROR]   ConstraintViolationsTest.shouldSupportMethodParameters:77->simulateValidation:109 » NoProviderFound
[ERROR] org.eclipse.krazo.util.AnnotationUtilsTest.getAnnotation(org.eclipse.krazo.util.AnnotationUtilsTest)
[ERROR]   Run 1: AnnotationUtilsTest.before:65 » ServiceConfiguration Error instantiating :org....
[ERROR]   Run 2: AnnotationUtilsTest.after:83 NullPointer
[INFO]
[ERROR] org.eclipse.krazo.util.AnnotationUtilsTest.getAnnotationOnMethod(org.eclipse.krazo.util.AnnotationUtilsTest)
[ERROR]   Run 1: AnnotationUtilsTest.before:65 » ServiceConfiguration Error instantiating :org....
[ERROR]   Run 2: AnnotationUtilsTest.after:83 NullPointer
[INFO]
[ERROR] org.eclipse.krazo.util.AnnotationUtilsTest.hasAnnotation(org.eclipse.krazo.util.AnnotationUtilsTest)
[ERROR]   Run 1: AnnotationUtilsTest.before:65 » ServiceConfiguration Error instantiating :org....
[ERROR]   Run 2: AnnotationUtilsTest.after:83 NullPointer
[INFO]
[ERROR] org.eclipse.krazo.util.AnnotationUtilsTest.hasAnnotationOnMethod(org.eclipse.krazo.util.AnnotationUtilsTest)
[ERROR]   Run 1: AnnotationUtilsTest.before:65 » ServiceConfiguration Error instantiating :org....
[ERROR]   Run 2: AnnotationUtilsTest.after:83 NullPointer
[INFO]
[INFO]
[ERROR] Tests run: 183, Failures: 0, Errors: 8, Skipped: 0


[INFO] Running org.eclipse.krazo.binding.validate.ConstraintViolationsMvcBindingTest
[ERROR] Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 0.005 s <<< FAILURE! - in org.eclipse.krazo.binding.validate.ConstraintViolationsMvcBindingTest
[ERROR] org.eclipse.krazo.binding.validate.ConstraintViolationsMvcBindingTest  Time elapsed: 0.005 s  <<< ERROR!
javax.validation.NoProviderFoundException: Unable to create a Configuration, because no Jakarta Bean Validation provider could be found. Add a provider like Hibernate Validator (RI) to your classpath.
        at org.eclipse.krazo.binding.validate.ConstraintViolationsMvcBindingTest.initValidator(ConstraintViolationsMvcBindingTest.java:60)

[INFO] Running org.eclipse.krazo.binding.validate.ConstraintViolationsTest
[ERROR] Tests run: 3, Failures: 0, Errors: 3, Skipped: 0, Time elapsed: 0.002 s <<< FAILURE! - in org.eclipse.krazo.binding.validate.ConstraintViolationsTest
[ERROR] shouldSupportMethodParameters(org.eclipse.krazo.binding.validate.ConstraintViolationsTest)  Time elapsed: 0 s  <<< ERROR!
javax.validation.NoProviderFoundException: Unable to create a Configuration, because no Jakarta Bean Validation provider could be found. Add a provider like Hibernate Validator (RI) to your classpath.
        at org.eclipse.krazo.binding.validate.ConstraintViolationsTest.simulateValidation(ConstraintViolationsTest.java:109)
        at org.eclipse.krazo.binding.validate.ConstraintViolationsTest.shouldSupportMethodParameters(ConstraintViolationsTest.java:77)

Reply:
Does this also occur when running the unmodified Krazo build? So without your changes?

Reply:
this stack trace is exactly when running unmodified Krazo build (master branch) test in my machine.